helm-frontend/src/components/LeaderboardTables.tsx

```diff
@@ -39,19 +37,7 @@ export default function LeaderboardTables({
   const [filteredModels, setFilteredModels] =
     useState<string[]>(modelsToFilter);
 
-  const [schema, setSchema] = useState<Schema | undefined>(undefined);
-
-  useEffect(() => {
-    const controller = new AbortController();
-    async function fetchData() {
-      const schema = await getSchema(controller.signal);
-      setSchema(schema);
-    }
-
-    void fetchData();
-    return () => controller.abort();
-  }, []);
-
+  /*
```

Comment:
Not sure if we want to keep this here or archive elsewhere, or remove entirely.

Reply:
Let's just remove this. We can always get it back from an old git commit.